.travis.yml

```diff
@@ -3,7 +3,10 @@
 # Instead use the files under the "/sample_files" directory for that purpose.
 language: python
 sudo: false
-cache: pip
```

Comment:
@pedrobaeza 
I updated next comment: https://github.com/OCA/maintainer-quality-tools/pull/257#issue-107654439
And check next issue: 
https://github.com/travis-ci/travis-ci/issues/4840